package.json

```diff
@@ -28,82 +28,82 @@
   ],
   "fontawesome": {
     "fal": [
-      "faSquare"
+      "square"
     ],
     "fas": [
-      "faAdjust",
-      "faCaretLeft",
-      "faCaretRight",
-      "faCheck",
-      "faCheckCircle",
-      "faCheckSquare",
-      "faChevronLeft",
-      "faChevronRight",
-      "faDotCircle",
-      "faExchange",
-      "faExclamationCircle",
-      "faFolder",
-      "faFolderOpen",
-      "faGripVertical",
-      "faInfoCircle",
-      "faKeyboard",
-      "faMinusSquare",
-      "faSignInAlt",
-      "faSignOutAlt",
-      "faTimes",
-      "faTimesCircle",
-      "faToggleOn"
+      "arrow-right-arrow-left",
```

Comment:
@nmajor25 now when we add font names they'll match the class we use in the template and what is copy/pasteable from fontawesome.com